packages/html/ds/src/table/table.html

```diff
@@ -0,0 +1,37 @@
+{% macro govieTable(props) %}
+  <table class="gi-table {{ props.className | default('') }}">
```

Comment:
are we still adding the `className` as a prop for components ?

Reply:
I don't see `className` in the schema, I think you can remove it